site/content/en/docs/commands/profile.md

```diff
@@ -89,6 +89,7 @@ minikube profile list [flags]
 ### Options
 
 ```
+  -l, --light           If true, only display vaild profiles and do not check status.
```

Comment:
if true, returns list of profiles faster by skipping validating the status of the cluster.